.github/workflows/clang-tidy-pr-comments.yaml

```diff
@@ -9,42 +9,53 @@ on:
 
 jobs:
   clang-tidy-pr-comments:
-    if: ${{ github.event.workflow_run.event == 'pull_request' && github.event.workflow_run.conclusion == 'success' }}
+    if: ${{ github.event.workflow_run.event == 'pull_request' && contains(fromJson('["success", "failure"]'), github.event.workflow_run.conclusion) }}
```

Comment:
FYI: https://docs.github.com/en/actions/learn-github-actions/expressions#example-matching-an-array-of-strings